nodejs/package.json

```diff
@@ -28,27 +28,28 @@
     "typescript": "4.1.3"
   },
   "dependencies": {
-    "@opentelemetry/sdk-trace-node": "1.0.1",
-    "@opentelemetry/api-metrics": "0.27.0",
-    "@opentelemetry/instrumentation": "0.27.0",
-    "@opentelemetry/instrumentation-aws-sdk": "~0.5.0",
-    "@opentelemetry/instrumentation-aws-lambda": "~0.28.0",
-    "@opentelemetry/instrumentation-dns": "~0.27.0",
-    "@opentelemetry/instrumentation-express": "~0.28.0",
-    "@opentelemetry/instrumentation-graphql": "~0.27.0",
-    "@opentelemetry/instrumentation-grpc": "~0.27.0",
-    "@opentelemetry/instrumentation-hapi": "~0.27.0",
-    "@opentelemetry/instrumentation-http": "~0.27.0",
-    "@opentelemetry/instrumentation-ioredis": "~0.28.0",
-    "@opentelemetry/instrumentation-koa": "~0.28.0",
-    "@opentelemetry/instrumentation-mongodb": "~0.28.0",
-    "@opentelemetry/instrumentation-mysql": "~0.28.0",
-    "@opentelemetry/instrumentation-net": "~0.27.0",
-    "@opentelemetry/instrumentation-pg": "~0.28.0",
-    "@opentelemetry/instrumentation-redis": "~0.28.0",
-    "@opentelemetry/resource-detector-aws": "^1.0.2",
-    "@opentelemetry/resources": "^1.0.1",
-    "@splunk/otel": "0.17.0",
+    "@opentelemetry/sdk-trace-node": "1.1.0",
+    "@opentelemetry/api": "1.1.0",
+    "@opentelemetry/api-metrics": "0.28.0",
+    "@opentelemetry/instrumentation": "0.28.0",
+    "@opentelemetry/instrumentation-aws-sdk": "0.7.0",
+    "@opentelemetry/instrumentation-aws-lambda": "0.28.0",
+    "@opentelemetry/instrumentation-dns": "0.28.0",
+    "@opentelemetry/instrumentation-express": "0.28.0",
+    "@opentelemetry/instrumentation-graphql": "0.28.0",
+    "@opentelemetry/instrumentation-grpc": "0.28.0",
+    "@opentelemetry/instrumentation-hapi": "0.28.0",
+    "@opentelemetry/instrumentation-http": "0.28.0",
+    "@opentelemetry/instrumentation-ioredis": "0.28.0",
+    "@opentelemetry/instrumentation-koa": "0.28.0",
+    "@opentelemetry/instrumentation-mongodb": "0.28.0",
+    "@opentelemetry/instrumentation-mysql": "0.28.0",
+    "@opentelemetry/instrumentation-net": "0.28.0",
+    "@opentelemetry/instrumentation-pg": "0.28.0",
+    "@opentelemetry/instrumentation-redis": "0.28.0",
+    "@opentelemetry/resource-detector-aws": "1.1.2",
```

Comment:
Actually a bunch of these are outdated. Should we upgrade?

Reply:
This is additional fallout from the node12 requirement.  When aws Lambda drops node12 support we will do another upgrade round.